addons/mail/static/src/messaging/widget/form_renderer/form_renderer.scss

```diff
@@ -0,0 +1,26 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_FormRenderer_chatterContainer {
+    display: flex;
+    flex: 1 1 auto;
+    margin: 0 auto;
+    max-width: 1140px;
+    padding: map-get($spacers, 3) map-get($spacers, 3) map-get($spacers, 5);
+    width: 100%;
+}
+
+/*
+    Specificity needed to override generic style made in
+    web_enterprise/static/src/scss/form_view.scss
+*/
+.o_form_view .o_form_sheet_bg .o_FormRenderer_chatterContainer {
+    margin-left: auto;
+    margin-right: auto;
+}
```

Comment:
This specificity is still not OK for me, especially when coming from enterprise. I haven't checked the enterprise part, but wasn't it possible to remove that part of the code from there?

Reply:
I can remove it easily, but as I said, it's difficult to check if anything else could be in `form_sheet_bg` and need the rule in form_view.scss with negative margins.